src/explore-education-statistics-admin/src/pages/release/components/ReleasePublishingCurrrentStatus.tsx

```diff
@@ -0,0 +1,48 @@
+import StatusBlock, { StatusBlockColors } from '@admin/components/StatusBlock';
+import { ReleaseStageStatuses } from '@admin/services/releaseService';
+import React from 'react';
+import Tag from '@common/components/Tag';
+
+const approvedStatuses = ['Complete', 'Scheduled'];
+
+interface Props {
+  color?: StatusBlockColors;
+  currentStatus?: ReleaseStageStatuses;
+  isApproved?: boolean;
+}
+
+const ReleasePublishingCurrentStatus = ({
```

Comment:
Minor, but would suggest naming this to `ReleasePublishingStatusTag` as it's a little clearer what this is compared to the `ReleasePublishingStatus` component.